### PR TITLE
Paella 7 matomo plugin

### PR DESCRIPTION
**Pull request by miesgre**
This PR  adds two new plugins:
- org.opencast.paella.matomo.userTrackingDataPlugin
  matomo usertracking plugin (will ask for cookie consent)

- org.opencast.paella.cookieconsent
  New button to modify the cookie consent

### Your pull request should…

* [x] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by mliradelc** [comment]
Thanks, I will review it!

File: etc/ui-config/mh_default_org/paella7/config.json [hunk 1]
```diff
@@ -19,6 +19,33 @@
...
+            "required": false
+        },
+        {
+            "type": "targeting",
+            "title": "Targeting and advertising cookies",
+            "description": "These cookies are used to show advertising that is likely to be of interest to you based on your browsing habits.",
+            "required": false
+        }
+    ],
+
```
Comment (mliradelc) [inline etc/ui-config/mh_default_org/paella7/config.json:48]: Can you add a little description about this in Paella player 7 opencast documentation?

**Review by mliradelc — CHANGES_REQUESTED** [review]
Works as expected. The cookie banner appears and is very complete.
Can you make some changes:
- Target branch from `develop` to `r/13.x`
- Add some documentation about this new feature

**Comment by miesgre** [comment]
> Works as expected. The cookie banner appears and is very complete. Can you make some changes:
> 
> * Target branch from `develop` to `r/13.x`
> * Add some documentation about this new feature

Done.

**Review by mliradelc — APPROVED** [review]
Wonderful! Works as expected

**Pull request merged**